data/lang/jp.json

```diff
@@ -250,7 +250,7 @@
   "data_startres" : "初登場F(再登場F)",
   "data_isboss" : "ボス",
   "data_boss" : "ボス",
-  "data_bossshake" : "ボス (衝撃波)",
+  "data_bossshake" : "ボス (Shake)",
```

Comment:
On `data_bossshake`, it's the one that shakes the battle. Every boss causes the `Shock wave` (This is what I got from translator), but this normal boss flag is used as `data_boss`